examples/chip-tool/commands/tests/TestCommand.cpp

```diff
@@ -20,30 +20,23 @@
 
 constexpr uint16_t kWaitDurationInSeconds = 30;
 
-CHIP_ERROR TestCommand::Run(PersistentStorage & storage, NodeId localId, NodeId remoteId)
+CHIP_ERROR TestCommand::Run(NodeId localId, NodeId remoteId)
 {
-    ReturnErrorOnFailure(mOpCredsIssuer.Initialize(storage));
+    CHIP_ERROR err = CHIP_NO_ERROR;
 
-    chip::Controller::CommissionerInitParams params;
+    {
+        chip::DeviceLayer::StackLock lock;
 
-    params.storageDelegate                = &storage;
-    params.operationalCredentialsDelegate = &mOpCredsIssuer;
+        err = GetExecContext()->commissioner->GetDevice(remoteId, &mDevice);
+        ReturnErrorOnFailure(err);
 
-    ReturnErrorOnFailure(mCommissioner.SetUdpListenPort(storage.GetListenPort()));
-    ReturnErrorOnFailure(mCommissioner.Init(localId, params));
-    ReturnErrorOnFailure(mCommissioner.ServiceEvents());
-    ReturnErrorOnFailure(mCommissioner.GetDevice(remoteId, &mDevice));
-
-    ReturnErrorOnFailure(NextTest());
+        err = NextTest();
+        ReturnErrorOnFailure(err);
+    }
 
     UpdateWaitForResponse(true);
```

Comment:
Again, this needs to happen before we unlock (either under the lock or before we take the lock).

Reply:
Will fix.